src/ledgerwriter/pom.xml.versionsBackup

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These *.versionsBackup files should not be checked in.

Reply:
Fixed.